modules/core/shared/src/main/scala/lucuma/core/math/geom/GmosOiwfsProbeArm.scala

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.math.geom
```

Comment:
We may want this to be `lucuma.core.geom`? The instrument definitions may not be exactly math.

Reply:
Yeah, maybe let's put all the geom stuff in `lucuma.core.geom` for now.